src/components/Global/Navbar.vue

```diff
@@ -22,7 +22,7 @@
       "
     >
       <!-- Dev Protocol Icon -->
-      <a href="https://devprotocol.xyz" class="text-gray-900 p-4">
+      <a href="https://devprotocol.xyz" class="text-gray-900 md:p-4 py-4">
```

Comment:
Note: This is the only real change I made don't mind the ones below, they're caused by a plugin in my editor.